core/new-gui/src/app/workspace/component/result-panel/result-panel.component.ts

```diff
@@ -64,7 +77,27 @@ export class ResultPanelComponent {
       value => this.showResultPanel = value,
     );
   }
-
+  handleHighLightOperator() : void {
+    const highlightedOperators = this.workflowActionService.getJointGraphWrapper().getCurrentHighlightedOperatorIDs();
```

Comment:
one more case you need to handle: if the highlighted operator length is 0 or >1, clear the result panel.